jupyter_server/extension/application.py

```diff
@@ -180,6 +190,10 @@ def get_extension_package(cls):
     def get_extension_point(cls):
         return cls.__module__
 
+    @classmethod
```

Comment:
i think "firewall" makes this sound a little more security-relevant than it is... perhaps "filter" or something...

Reply:
Good point I will change the name